nemo/collections/nlp/models/machine_translation/megatron_nmt_model.py

```diff
@@ -91,6 +110,11 @@ def setup(self, stage=None):
         self.setup_validation_data(self._cfg.validation_ds)
         self.setup_test_data(self._cfg.test_ds)
 
+        # when using pipeline model parallel the final stage need to initialize word embeddings
+        if parallel_state.get_pipeline_model_parallel_world_size() > 1:
+            self.enc_dec_model.sync_initial_word_embeddings()
```

Comment:
This looks like it should be shared between all token level megatron models, right?